sim/common/vanilla/item_effects.go

```diff
@@ -2457,6 +2459,55 @@ func init() {
 		})
 	})
 
+	// https://www.wowhead.com/classic/item=233627/jom-gabbar
+	// Use: Increases attack power by 70 and an additional 70 every 2 sec.  Lasts 20 sec. (2 Min Cooldown)
+	core.NewItemEffect(JomGabbar, func(agent core.Agent) {
+		character := agent.GetCharacter()
+		actionID := core.ActionID{SpellID: 1213366}
+		jomGabbarAura := character.GetOrRegisterAura(core.Aura{
+			Label:    "Jom Gabbar",
+			ActionID: actionID,
+			Duration: time.Second*20,
+			OnGain: func(aura *core.Aura, sim *core.Simulation) {
+				core.StartPeriodicAction(sim, core.PeriodicActionOptions{
+					Period:   time.Second * 2,
+					NumTicks: 10,
+					Priority: core.ActionPriorityAuto,
+					TickImmediately: true,
+					OnAction: func(sim *core.Simulation) {
+						character.AddStatDynamic(sim, stats.AttackPower, 70)
+						character.AddStatDynamic(sim, stats.RangedAttackPower, 70)
+					},
+				})
+			},
+			OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+				character.AddStatDynamic(sim, stats.AttackPower, -700)
+				character.AddStatDynamic(sim, stats.RangedAttackPower, -700)
+			},
+		})		
```

Comment:
We talked about this on Discord but just documenting it here, can we make this a stackable aura so that users can more easily keep track of the stacking AP bonus? You would add a stack during the periodic ticks, remove the `OnExpire`, and use `OnStacksChanged` to modify stats